src/document-settings/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { PluginPostStatusInfo } from '@wordpress/edit-post';
+import { registerPlugin } from '@wordpress/plugins';
+import { FormToggle } from '@wordpress/components';
+
+const PostStatusElement = ( { hasDisabledPopups, onChange } ) => {
+	return (
```

Comment:
@claudiulodro built a similar feature in Newspack Ads in https://github.com/Automattic/newspack-ads/pull/26. For consistency's sake could we match the UI style?

<img width="1535" alt="Screen Shot 2020-04-23 at 7 33 03 AM" src="https://user-images.githubusercontent.com/1477002/80094872-0319ba00-8535-11ea-9867-3bd693674eaf.png">

Reply:
done: 68f695404cd3d1d24f7e00b804e0951d54825968